pyproject.toml

```diff
@@ -51,8 +51,14 @@ test = [
 # All packages
 all = ["ambersim[dev, test]"]
 
+[project.scripts]
+install-mujoco-from-src = "ambersim._scripts._install_shim:entrypoint"
```

Comment:
I believe this is fixed after hacking at it for a bit. Main change: the default `mujoco` installation path is now `$HOME/mujoco`, so we should recommend that people specify a reasonable local path.

The main reason I don't want to exclude the script is because `ambersim` might be slightly unstable and depend on bleeding edge updates. It's looking like they might wait ~1 month between releases, so if there's some feature that's really good, we might be out of luck if this script isn't packaged with the `ambersim` install. I think keeping depending packages synced up with it is in general good (or at least providing that option). Also despite appearances, it's not actually that hard to install the script correctly - I just had to fiddle with some paths that don't generalize well.

Reply:
Sounds good. Just tested on a fresh env in my machine and it seems to work now.